### ISSUE TITLE
[BUG](sort of. exclusive to Synitra users)

### ISSUE THREAD

**Issue by Machinatory** [DaFuqs/Spectrum#452]
Synitra users heads up. trinkets are dead in the water, leaving them unusable. not curios, not trinkets slots. i'm not about to assume you have the time for it, so i'm pointing this out until we can figure out a solution

**Comment by Gronglegrowth** [comment]
What do you mean by this?

**Comment by Robotgiggle** [comment]
If this is the bug where the trinkets don't work and also vanish on relog, try dying, relogging before putting anything back on, and _then_ putting them back on. You'll also need to relog every time you die, before putting any of the trinkets back on.

**Comment by luo-zi** [comment]
> Synitra users heads up. trinkets are dead in the water, leaving them unusable. not curios, not trinkets slots. i'm not about to assume you have the time for it, so i'm pointing this out until we can figure out a solutionSynitra用户注意了。小饰品在水里死了，让它们无法使用。不是古董也不是小饰品我不想假设你有时间，所以我指出这一点，直到我们找到一个解决方案

did you means "Sinytra Connector" mod

**Comment by Machinatory** [comment]
no, it's even worse. trinkets don't even go into the slot, period. rc, directly trying to put them in the slot, etc.

**Comment by Machinatory** [comment]
> > Synitra users heads up. trinkets are dead in the water, leaving them unusable. not curios, not trinkets slots. i'm not about to assume you have the time for it, so i'm pointing this out until we can figure out a solutionSynitra用户注意了。小饰品在水里死了，让它们无法使用。不是古董也不是小饰品我不想假设你有时间，所以我指出这一点，直到我们找到一个解决方案
> 
> did you means "Sinytra Connector" mod

yes

**Comment by Gronglegrowth** [comment]
> no, it's even worse. trinkets don't even go into the slot, period. rc, directly trying to put them in the slot, etc.

Do you have the required progression? Trinkets are progress gated; that is you cannot equip them until you have unlocked them.

**Comment by f-raZ0R** [comment]
> > no, it's even worse. trinkets don't even go into the slot, period. rc, directly trying to put them in the slot, etc.
> 
> Do you have the required progression? Trinkets are progress gated; that is you cannot equip them until you have unlocked them.

Any updates on this?
Would like more information about this issue with trinkets

**Comment by ALEILGOD** [comment]
trinkets is ok on connector, i tested it. Try the command /advancement grant (your username) everything or something like that and all the trinkets should have the trinket slot (I TRIED WITH THE COMPAT LAYER FOR ACCESSORIES) @DaFuqs close the issue

**Comment by imreallybadatnames** [comment]
what

**Comment by ALEILGOD** [comment]
It works. fixed.

**Comment by ALEILGOD** [comment]
i tried it

**Comment by ALEILGOD** [comment]
I have connectored Spectrum 1.8.3 with standalone Cardinal Componets API ant the trinkets compat layer for the mod Accessories and everything works

**Comment by DaFuqs** [comment]
Works now!
